autonomy/analyse/abci/logs.py

```diff
@@ -109,5 +109,5 @@ def parse_file(  # pylint: disable=too-many-locals
             )
 
     print("\nERRORS:")
-    for error in errors:
+    for error in errors:  # pragma: nocover
```

Comment:
Same goes with this function, I think David V would be more suitable to write tests for this method.